polynote-kernel/src/main/scala-2.13/polynote/kernel/util/KernelReporter.scala

```diff
@@ -0,0 +1,81 @@
+package polynote.kernel.util
+
+import cats.data.Ior
+import polynote.kernel.{CompileErrors, KernelReport, Pos}
+
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import scala.reflect.internal.util.Position
+import scala.tools.nsc.Settings
+import scala.tools.nsc.reporters.FilteringReporter
+
+case class KernelReporter(settings: Settings) extends FilteringReporter {
```

Comment:
Might be worth a quick comment describing why we need a different definition for Scala 2.13? It would help me, at least :)

Reply:
The API of `Reporter` changed in Scala 2.13.

Reply:
I'll add a comment to this effect.

Reply:
Added an explanatory comment here